ingest.py

```
@@ -0,0 +1,65 @@
+# ingest.py
+
+from pathlib import Path
+import docx
+import fitz  # PyMuPDF
+import pandas as pd
+
+def process_docx(file_path):
```

Ruff (0.12.2)

Missing return type annotation for public function `process_docx`

(ANN201)